test/components/form-attachment/list.spec.js

```diff
@@ -1055,11 +1071,8 @@ describe('FormAttachmentList', () => {
 
     describe('Datasets preview hint', () => {
       beforeEach(() => {
-        testData.extendedProjects.createPast(1, {
-          name: 'My Project Name',
-          forms: 1,
-          datasets: 1
-        });
+        testData.extendedProjects.createPast(1, { forms: 2, datasets: 1 });
```

Comment:
I changed `forms` from 1 to 2 with the thought that if there's already an entity list, then there should also be another form in addition to the draft form. I don't think this change has practical ramifications, but I think it's slightly clearer.

I also removed the project name, which isn't used in the test.